lib/pages/pool/MyLiqudity/MyLiquidity.dart

```diff
@@ -693,6 +693,7 @@ class _MyLiquidityState extends State<MyLiquidity> {
                                         onPressed: () {
                                           setState(() {
                                             currentTabIndex = 0;
+                                            value = 0;
```

Comment:
We need to update Pool Controller as well here. You can reference what I wrote on "value" change.